RedHat-UX/red-hat-design-system
# fix: rh-footer equal columns

## What I did

1.  Fixes #442 


## Testing Instructions

- [ ] Verify that the columns support long text and maintain equal columns widths
- git fetch && git checkout fix/footer-equal-columns
- npm run dev
- Visit http://localhost:8001/demo/rh-footer/
- Using the dev tools, make one of the links contain very long text.  The columns should remain equal widths on desktop.

## Notes to Reviewers

- I did a quick googling to verify that the grid-auto-flow properties are supported in all evergreen browsers.
